katsdpcontroller/generator.py

```diff
@@ -38,7 +39,7 @@
     'katsdpcontim',
     'katsdpdisp',
     'katsdpimager',
-    'katsdpingest_titanx',
+    'katsdpingest_' + INGEST_GPU_SHORTNAME,
```

Comment:
Yeah, it would simplify life in the future, at the expense of some complication now to make sure that all the pieces make the change in lock-step and/or we provide the image under both names. I think I'd rather sort that out after merging split-controllers since at that point we are better able to keep the bits in sync for a given subarray-product. I've opened SPR1-248.